rmgpy/molecule/molecule.py

```diff
@@ -2618,6 +2618,152 @@ def enumerate_bonds(self):
+                        bond.increment_order()
+                        atom0.decrement_radical()
+                        atom1.decrement_radical()
+                        desorbed_molecules.append(dummy_molecule.copy(deep=True))
+                        if (atom0.radical_electrons and
+                                atom1.radical_electrons and
+                                bond.order < 3):
+                            # There are still spare adjacenct radicals, so do it again
+                            bond.increment_order()
+                            atom0.decrement_radical()
+                            atom1.decrement_radical()
+                            desorbed_molecules.append(dummy_molecule.copy(deep=True))
+                        if (atom0.lone_pairs and
+                                atom1.lone_pairs and 
+                                bond.order < 3):
+                            # X#C-C#X will end up with .:C-C:. in gas phase
+                            # and we want to get to .C#C. but not :C=C:
+                            bond.increment_order()
+                            atom0.decrement_lone_pairs()
+                            atom0.increment_radical()
+                            atom1.decrement_lone_pairs()
+                            atom1.increment_radical()
+                            desorbed_molecules.append(dummy_molecule.copy(deep=True))
+                    #For bidentate CO because we want C[-1]#O[+1] but not .C#O.
+                    if (bond.order == 3 and atom0.radical_electrons and 
+                        atom1.radical_electrons and 
+                        (atom0.lone_pairs or atom1.lone_pairs)):
+                        atom0.decrement_radical()
+                        atom1.decrement_radical()
+                        if atom0.lone_pairs:
+                            atom1.increment_lone_pairs()
+                        else:
+                            atom0.increment_lone_pairs()
+                        desorbed_molecules.append(dummy_molecule.copy(deep=True))
+
+        new_mols = []
+        for dummy_molecule in desorbed_molecules:
+            try:
+                dummy_molecule.get_labeled_atoms('*2')
+            except ValueError:
+                # No *2 atoms
+                continue
+            new_mol = dummy_molecule.copy(deep=True)
+            for atom in new_mol.get_labeled_atoms('*2'):
+                if atom.radical_electrons == 2:
+                    atom.decrement_radical()
+                    atom.decrement_radical()
+                    atom.increment_lone_pairs()
+                    new_mols.append(new_mol.copy(deep=True))
+        desorbed_molecules.extend(new_mols)
+
+        for dummy_molecule in desorbed_molecules[:]:
+            try:
+                dummy_molecule.update_connectivity_values()
+                dummy_molecule.update()
+            except:
```

Comment:
Can we be more specific with what exceptions we catch? I'm not sure what the update methods throw. The "LGTM" bot will probably complain about this.

Reply:
I added `AtomTypeError` here since I believe this is the only error we might expect